charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -200,7 +194,7 @@ spec:
         {{ end }}
 
         image: budibase/apps:{{ .Values.globals.appVersion | default .Chart.AppVersion }}
-        imagePullPolicy: Always
```

Comment:
This might break QA because we push a `develop` tag there - and a rollout restart won't pull the latest then. Tbf we should just fix that

Reply:
Whoops, good catch. I changed this to make iterating on my own cluster faster. Put it back to how it was. 🙏